tests/components/generic/conftest.py

```diff
@@ -120,3 +121,10 @@ async def setup_entry(hass, config_entry):
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
     return config_entry
+
+
+@pytest.fixture(autouse=True)
+async def setup_stream(hass):
```

Comment:
Can this instead setup the integration to achieve this? That seems like the more canonical way to do this type of thing for normal integrations, though I know in stream we have a lot of more traditional unit tests like this, generally other integration tests prefer to use the actual integration APIs instead.

Reply:
I believe so, let's try that.

Reply:
Actually, this can just be removed as we don't use convert_stream_options here after https://github.com/home-assistant/core/pull/73237